runelite-client/src/main/java/net/runelite/client/plugins/zalcano/ZalcanoPlugin.java

```diff
@@ -86,6 +93,9 @@
 	@Getter(AccessLevel.PACKAGE)
 	private NPC golem;
 
+	@Getter(AccessLevel.PACKAGE)
+	private final List<Player> Players = new ArrayList<>();
```

Comment:
This field (which should NOT be capitalized btw) is really really useless and inefficient.

I would tell you to use a Set instead of a List to begin with, but I think even that's a bit too much. Rather than listening for every player coming and going and keeping track of them all, why not use client.getPlayers once every <insert reasonable refresh rate here> and count them then? It's not like this would be wildly accurate to begin with

Reply:
Added.

Reply:
Now to test it.

Reply:
Works fine as far i can tell.